nucleus/common/internal-api/src/main/java/org/glassfish/internal/deployment/analysis/DeploymentSpan.java

```diff
@@ -115,9 +117,15 @@ public String toString() {
         if (f != null) {
             NS_FORMATTER.formatTo(f, sb);
         }
-        sb.append('\t');
+        sb.append('\t')
+            .append(getNestLevel())
+            .append('\t');
         if (context != null) {
-            sb.append(context);
+            sb.append(context.getLevelName(TraceContext.Level.CONTAINER))
+                .append('\t')
+                .append(context);
+        } else {
+            sb.append('\t');
```

Comment:
Is tab really the right way to do this? I think the current output example shows that it mostly lines up but sometimes it is 1-2 tabs out of alignment. Wouldn't use of format with fixed with spacing be more accurate. Also for things like missing "columns" like the case here where context sometimes is empty/non existent?

Reply:
Tabs import well into spreadsheet, which is the currently used "frontend' for the feature. The output is too complex to just inspect visually in the log, so it's adapted for import by some tooling.